abstractions/php/src/AuthenticationProvider.php

```diff
@@ -3,6 +3,6 @@
 
 use Http\Promise\Promise;
 
-interface AuthenticationProviderInterface {
+interface AuthenticationProvider {
```

Comment:
FYI the prototype for this interface has evolved to support multiple authentication schemes https://github.com/microsoft/kiota/pull/498